_data/apps.yml

```diff
@@ -2258,4 +2258,13 @@
   website: "https://colibri.opqr.co"
   keywords:
     - "browser"
-  icon: "colibri.png"
\ No newline at end of file
+  icon: "colibri.png"
+-
+  name: "itch"
+  description: "The itch.io app, itch, lets you effortlessly download and run games and software from itch.io."
```

Comment:
This description seems overly "itchy". Maybe it could be shortened? How about this:

> Download and run games and software from itch.io